framework/config/src/main/java/org/apache/cloudstack/framework/config/RetrieveDiagnosticsDaoImpl.java

```diff
@@ -31,14 +31,14 @@
 @Component
 public class RetrieveDiagnosticsDaoImpl extends GenericDaoBase<RetrieveDiagnosticsVO, String> implements RetrieveDiagnosticsDao
 {
-    private final SearchBuilder<RetrieveDiagnosticsVO> DiagnosticsSearchByType;
+//    private final SearchBuilder<RetrieveDiagnosticsVO> DiagnosticsSearchByType;
```

Comment:
did it work, without? (just remove, git will find it back if you need it;)

Reply:
Yes, it did work thanks